tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -374,3 +382,131 @@ def bgp_title_pattern_munge(tests, id, provider=nil)
   t[:vrf] = 'default' if t[:vrf].nil?
   t
 end
+
+# setup_mt_full_env
+# Check and set up prerequisites for Multi-Tenancy Full (MT-full) testing.
+# MT-full currently requires an F3 line module. This method will update
+# the tests hash with the names of the default vdc and also an appropriate
+# test interface name to use for testing.
+# tests[:vdc] The default vdc name
+# tests[:intf] A compatible interface to use for MT-full testing.
+def setup_mt_full_env(tests, testcase)
+  # MT-full tests require a specific linecard. Search for a compatible
+  # module and enable it.
+
+  testheader = tests[:testheader]
+  mod = 'f3'
+  step 'Check for Compatible Line Module' do
+    tests[:intf] = mt_full_interface
+    break if tests[:intf]
+    prereq_skip(testheader, testcase,
+                "MT-full tests require #{mod} or compatible line module")
+  end if tests[:intf].nil?
```

Comment:
It allows me to completely bypass this `step` if the caller provides `tests[:intf]` instead of having this method figure it out. It's only marginally useful but I found it handy during troubleshooting. 

However, on second look I think it would be useful to add it to the VDC step because that would allow the caller to specify a non-default VDC, so I'll add it and remove the 'default' references from the messages.